### DISCUSSION TITLE
visualizer issues

### DISCUSSION THREAD

**Discussion by Gonzose** [winder/Universal-G-Code-Sender#2162] [Help]
In the visualizer there is the white cone that should be aligned with the cursor.  In my UGS the white cone is about 2 inches off and does not align.  Makes it impossible to check coordinates.  
Anyone have any ideas how to solve this issue?  Is it calibration? Is it a glitch? 
BTW the yellow cone is set perfectly to the piece and bit location... the problem is just the white cone.
I've tried deleting the entire program and starting over but that hasn't helped. (could be I didn't get it entirely removed too)
Appreciate any help you can provide.

**Comment by breiler** [comment]
Can you post a screenshot?

**Reply by Gonzose** [reply]
![0222231026a_HDR](https://user-images.githubusercontent.com/125605897/220691809-2559634d-ef7f-43ea-9e65-3373c4287346.jpg)

**Reply by breiler** [reply]
Yes, something is not right...
These actions should be greyed out when disconnected and/or if there is no file loaded:
<img width="112" alt="image" src="https://user-images.githubusercontent.com/8962024/220704939-8a5c922d-7e41-4426-ab89-c5d776800dce.png">

The grey cone should not be shown at all when the mouse cursor is outside the grid:
<img width="214" alt="image" src="https://user-images.githubusercontent.com/8962024/220705455-bd9e1128-50e8-4d8a-8b3b-f3ced3b54fdd.png">

Can you post the logs, it is located in `C:/Users/[your username>]/.ugsplatform/2.X/dev/var/log/messages.log` or `C:/Users/[your username>]/AppData/Roaming/ugsplatform/var/log/messages.log` depending on which version of UGS you are running.

**Comment by Gonzose** [comment]
not finding files that match what you're asking for

**Reply by breiler** [reply]
The directory `C:/Users/[your username>]/AppData` might be hidden. If you open the file explorer and select the option to show hidden files and folders you should be able to see it.

**Comment by Gonzose** [comment]


**Reply by Gonzose** [reply]
hope thats what you're looking for.

**Reply by breiler** [reply]
Yes, unfortunatley it didn't contain any error messages related to the problems you are describing. 🤷‍♂️

**Reply by Gonzose** [reply]
looks like I found the culprit to this agonizing problem.  Turns out it was the display I added a second screen today and the display mode on laptop screen is 1920x1080, 60Hz 6-bit while the second screen display is 1366x768, 59.72 Hz 8-bit. Works perfect on the second screen display mode.
